modules/nf-core/quilt/quilt/meta.yml

```diff
@@ -0,0 +1,99 @@
+name: "quilt_quilt"
+description: QUILT is an R and C++ program for rapid genotype imputation from low-coverage sequence using a large reference panel.
+keywords:
+  - imputation
+  - low-coverage
+  - genotype
+  - genomics
+  - vcf
+tools:
+  - "quilt":
+      description: "Read aware low coverage whole genome sequence imputation from a reference panel"
+      homepage: "https://github.com/rwdavies/quilt"
+      documentation: "https://github.com/rwdavies/quilt"
+      tool_dev_url: "https://github.com/rwdavies/quilt"
+      doi: "10.1038/s41588-021-00877-0"
+      licence: "['GPL v3']"
+
+input:
```

Comment:
Can you order these in the same order as the input to the tool, with the meta2 and meta3 as well please.

Reply:
Yes, thank you for your suggestion.